src/components/customSearch/CustomSearchHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import { Grid, Link, Stack, Typography } from '@mui/material'
+
+const CustomSearchHeader = () => {
+  return (
+    <Grid item container direction="row" justifyContent="center" width="100%">
+      <Grid
+        item
+        container
+        width={'756px'}
+        direction={'row'}
+        justifyContent={'flex-start'}
+        sx={{ mt: 1, mb: 3 }}
+      >
+        <Stack direction={'column'} spacing={1}>
+          <Typography variant="h5">Custom Your Search Rules</Typography>
+          <Typography variant="body2">
+            <Link href={'/search/simple'}>Use simple search</Link>
```

Comment:
I don't know what the design is suggesting but we need to be consistent I think. Either 1st letter is capital, or every word has a capital letter.

Reply:
Designs are suggesting that titles should have a capital letter for each word and normal texts (e.g. descriptions) should have the first letter capital.
![image](https://user-images.githubusercontent.com/22627035/225274864-fa6fcba8-5d90-4bd1-b75f-f8f96a1e64da.png)